src/crash_context/mod.rs

```diff
@@ -37,7 +38,8 @@ pub mod imp;
 pub mod imp;
 
 #[cfg(target_arch = "aarch64")]
-pub type fpstate_t = libc::fpsimd_context; // Currently not part of libc! This will produce an error.
+#[allow(non_camel_case_types)]
+pub type fpstate_t = libc_user_fpsimd_struct; // Currently not part of libc! This will produce an error.
```

Comment:
Since you created your own struct, this will not produce an error anymore? So the second half of the comment can be removed, I think.